src/runtime_src/xdp/profile/device/device_trace_offload.h

```diff
@@ -40,57 +41,86 @@ enum class DeviceOffloadStatus {
 #define debug_stream \
 if(!m_debug); else std::cout
 
-class OclDeviceOffload {
+class DeviceTraceOffload {
 public:
-    OclDeviceOffload(xdp::DeviceIntf* dInt, std::shared_ptr<RTProfile> ProfileMgr,
+    XDP_EXPORT
+    DeviceTraceOffload(xdp::DeviceIntf* dInt, std::shared_ptr<RTProfile> ProfileMgr,
                      const std::string& device_name, const std::string& binary_name,
                      uint64_t offload_sleep_ms, uint64_t trbuf_sz,
                      bool start_thread = true);
-    ~OclDeviceOffload();
+    XDP_EXPORT
+    ~DeviceTraceOffload();
+    XDP_EXPORT
     void offload_device_continuous();
+    XDP_EXPORT
     bool should_continue();
+    XDP_EXPORT
     void start_offload();
+    XDP_EXPORT
     void stop_offload();
 
 public:
+    XDP_EXPORT
+    bool read_trace_init();
+    XDP_EXPORT
+    void read_trace_end();
+
+public:
+    XDP_EXPORT
     void set_trbuf_alloc_sz(uint64_t sz) {
         m_trbuf_alloc_sz = sz;
     };
+    XDP_EXPORT
```

Comment:
Export keyword is not needed for methods declared in header files. Please add the change.

Reply:
removed exports